### DISCUSSION TITLE
Minimal examples with mv3 (Google Chrome manifest 3)?

### DISCUSSION THREAD

**Discussion by valery-uga** [InboxSDK/InboxSDK#1217] [Q&A]
The page https://github.com/InboxSDK contains several examples for getting started with InboxSDK, which worked well for me to write an mv2 Chrome extension.

Is there a similar page for mv3 applications? Some minimal code to which I could add the functionality of my extension (TeX for Gmail)?

I am having a lot of trouble upgrading my extension to mv3.

**Comment by Macil** [comment]
The example at https://github.com/InboxSDK/hello-world uses mv3.
